docs/docs/get_started.md

```diff
@@ -0,0 +1,129 @@
+# arlas_playground
+
+## Clone the project
+
+The **arlas_playground** project contains scripts and resources you need to run the tutorials.
+
+Clone it in your working directory
+
+```shell
+git clone git@github.com:gisaia/arlas_playground.git
+cd arlas_playground
+```
+
+## Prerequisite
+
```

Comment:
in fact, we should have two cases:
- with arlas exploration stack
- with arlas cloud

So **Prerequisite** should be updated accordingly

Reply:
Let me know if it's clearer with the precisions

Reply:
I don't think you pushed the precisions yet